db/migrate/20200511102124_create_todo_items.rb

```diff
@@ -2,6 +2,7 @@ class CreateTodoItems < ActiveRecord::Migration[5.2]
   def change
     create_table :todo_items do |t|
       t.string :content
+      t.integer :priotity
```

Comment:
Have you written a corresponding migration to remove this column, I'm assuming this was a type and should be removed since i can see you've added a separate migration to add column priority

Reply:
Yeah, that should have been done. I'll fix it.